src/test/java/steps/SpaceStep.java

```diff
@@ -20,8 +21,17 @@
  * @author Jesus Menacho
  * @version 1.0
  */
-public class StarStep {
-    private StarPage starPage;
+public class SpaceStep {
+    private SpacePage starPage;
+    private Context context;
+
+    /**
+     * Constructor class.
+     * @param context type Context class.
+     */
+    public SpaceStep(Context context) {
+        this.context = context;
+    }
 
     /**
      * This method let create in the web page new name space.
```

Comment:
Please change the documentation according to our Code Conventions.

Reply:
Done.